crates/util/cometindex/src/index.rs

```diff
@@ -0,0 +1,20 @@
+use async_trait::async_trait;
+use sqlx::{Postgres, Transaction};
+
+use crate::ContextualizedEvent;
+
+pub type PgTransaction<'a> = Transaction<'a, Postgres>;
+
+/// Represents a specific index of raw event data.
+#[async_trait]
+pub trait Index: std::fmt::Debug {
```

Comment:
Maybe this should be called `AppIndex` or something?

Reply:
during in-air conversation we decided upon `AppView`.

31949e5b3 performs this change :bouquet: